domains/minhazurhossain.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "hello makal..",
+    "repo": "https://github.com/Minhazurhossain/minhazurhossain",
+    "owner": {
+        "username": "Minhazurhossain",
+        "email": "minhazurhossain98@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://minhazurhossain.github.io/resume/"
```

CNAME cannot have https or /
```suggestion
        "CNAME": "minhazurhossain.github.io"
```